spec/features/users/delete_user_spec.rb

```diff
@@ -16,7 +16,7 @@
       within row do
         find_link('Delete').click
       end
-
+      sleep 1.second
```

Comment:
Is it possible to use something like this: https://stackoverflow.com/a/21342673

This is using an `expect` to check for something (in the example, some CSS) on the new page after it has loaded. From some similar Stack Overflow pages it appears that the `have_css` function has a `wait` option to specify a timeout.

Reply:
thank you, i found accept_alert to suffice after researching your suggestion. It seems to work well and has a wait  option